modules/broker-cli-client/src/main/java/io/ballerina/messaging/broker/client/output/TableFormatter.java

```
@@ -138,43 +136,114 @@ public void printExchangeBindings(Binding[] bindings) {
         for (Binding binding : bindings) {
-            OUT_STREAM.printf(printTemplate, binding.getQueueName(), binding.getBindingPattern());
+            tempBindings.add(new String[]{binding.getQueueName(), binding.getBindingPattern()});
         }
+
+        printTable(new String[]{Binding.QUEUE_NAME, Binding.BINDING_PATTERN}, tempBindings);
     }
 
     @Override
     public void printConsumers(Consumer[] consumers) {
         if (consumers.length == 0) {
             return;
         }
-        int maxIdLength = Arrays.stream(consumers)
-                                .mapToInt(consumer -> String.valueOf(consumer.getId()).length())
-                                .max()
-                                .getAsInt();
-
-        int maxColumnSize = Math.max(maxIdLength, Consumer.CONSUMER_ID.length());
 
-        String printTemplate = "%-" + (maxColumnSize + TABLE_PADDING) + "s%-12s%s\n";
+        ArrayList<String[]> tempConsumers = new ArrayList<>();
 
-        OUT_STREAM.printf(printTemplate, Consumer.CONSUMER_ID, Consumer.IS_EXCLUSIVE, Consumer.FLOW_ENABLED);
         for (Consumer consumer : consumers) {
-            OUT_STREAM.printf(printTemplate, consumer.getId(), consumer.isExclusive(), consumer.isFlowEnabled());
+            tempConsumers.add(new String[]{Integer.toString(consumer.getId()),
+                                           Boolean.toString(consumer.isExclusive()),
+                                           Boolean.toString(consumer.isFlowEnabled())});
         }
+
+        printTable(new String[]{Consumer.CONSUMER_ID, Consumer.IS_EXCLUSIVE, Consumer.FLOW_ENABLED}, tempConsumers);
+
     }
 
     @Override
     public String toString() {
         return FORMATTER_NAME;
     }
+
+
+    /**
+     * This methods print given set of values in a table format.
+     *
+     * @param titles titles of each column of the table
+     * @param rows   values in rows
+     */
+    private void printTable(String[] titles, List<String[]> rows) {
```

Should we throw an error if the titles.length != rows.length?